src/lein_monolith/dependency.clj

```diff
@@ -27,6 +27,21 @@
     (condense-name (symbol (:group project) (:name project)))))
 
 
+(defn resolve-name
+  "Given a set of valid project names, determine the match for the named
+  project. This can be used to resolve the short name (meaning, no namespace)
+  to a fully-qualified project name. Returns a resolved key from
+  `project-names`, or nil if the resolution fails."
+  [project-names sym]
+  (let [valid-keys (set project-names)]
+    (cond
+      (valid-keys sym) sym
+      (valid-keys (condense-name sym)) (condense-name sym)
+      (nil? (namespace sym))
+        (first (filter #(= (name %) (name sym)) valid-keys))
```

Comment:
Shouldn't this throw a "couldn't resolve to just one project" error if more than one result appears here?